lib/publiccloud/utils.pm

```diff
@@ -109,7 +109,7 @@ sub registercloudguest {
     my ($instance) = @_;
     my $regcode = get_required_var('SCC_REGCODE');
     my $path = is_sle('>15') && is_sle('<15-SP3') ? '/usr/sbin/' : '';
-    my $suseconnect = $path . get_var("PUBLIC_CLOUD_SCC_ENDPOINT", (is_transactional) ? "transactional-update register" : "registercloudguest");
+    my $suseconnect = $path . get_var("PUBLIC_CLOUD_SCC_ENDPOINT", "registercloudguest");
```

Comment:
Also for older versions of SLEM?

Reply:
yes, transactional registration is a recent option I introduced during setup of new PC SL-micro <6.0 group [PR](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18951), but in daily runs was noted that this exception is not needed, working registercloudguest in any case.

Reply:
Ok, thanks for the clarification.